fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -67,7 +67,11 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :comment,
                                        env_name: "FL_TESTFAIRY_COMMENT",
                                        description: "Additional release notes for this upload. This text will be added to email notifications",
-                                       default_value: 'No comment provided') # the default value if the user didn't provide one
+                                       default_value: 'No comment provided'), # the default value if the user didn't provide one
+          FastlaneCore::ConfigItem.new(key: :testers_groups,
+                                       env_name: "FL_TESTFAIRY_TESTERS_GROUPS",
+                                       description: "Comma seperated list of tester groups to be notified",
+                                       default_value: "") # the default value is an empty list
```

Comment:
It would be good to represent this as an Array type option if it can take multiple values.

```ruby
type: Array,
default_value: []
```

Then, you can use `join(',')` to pass the values as a single string